scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#294.sql

```diff
@@ -0,0 +1,26 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- Change releases with positive latitudes
+UPDATE animal_release SET capture_location = ST_POINT(ST_X(capture_location), ST_Y(capture_location) * (-1))
+WHERE ST_Y(capture_location) >= 0;
+UPDATE animal_release SET release_location = ST_POINT(ST_X(release_location), ST_Y(release_location) * (-1))
```

Comment:
I think operator precedence is respected in sql, eg. 
```
postgres=# select 2 + 3 * -4
postgres-# ;
 ?column? 
----------
      -10
(1 row)
```

I am not really fussed, just wondering if there was a reason.